python/PEDvPUDplot.py

```diff
@@ -2,104 +2,166 @@
 """
 Created on Sat Nov  7 12:59:47 2015
 
-@author: ymseah
+Make scatterplot of pairwise uncorrected vs evolutionary distances.
+
+Pairwise uncorrected distances are obtained from SDT 1.0 (filename_mat.txt), while pairwise maximum likelihood distances are obtained from Tree-Puzzle 5.2 (outdist.txt). Both files are read to extract sequence ID and distance values.
+
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
-import itertools # for zipping, less memory consumption than zip
+import itertools
 
 filepath = raw_input("Enter path to SDT matrix file: ")
 filepath2 = raw_input("Enter path to TreePuzzle distance output file: ")
 
-# function to get sequence IDs from SDT matrix output file
 def getSDTid(filepath):
+    """
+    Reads SDT 1.0 matrix output (filename_mat.txt) and returns list of sequence IDs.
+    
+    ::param filepath::  Path to SDT file (input).
+    ::param sdtfile::   Input file.
```

Comment:
Sorry, should've made this clearer.  You don't need to document every variable that you use in this function.  You only need to document the parameters you pass into the function and what gets returned.  So for this all you need to document are the following parameters:
::param filepath::
::returns:: (If the function returns something, you want to say what it returns.)

You only want to document items that could be possibly be used by others.  So developers are usually just interested in using your code, so telling them what they need to pass and what they get back is enough.  If they want to read your code to understand it better, your code should already be self-documenting.